<launch>
  <node pkg="mqtt_ros_bridge" type="imu_bridge.py" name="imu_bridge_left_hand" output="screen">
    <remap from="/imu_data" to="/imu_left_hand"/>
    <param name="device_name" type="string" value="G Watch R 5509" />
  </node>
</launch>

